lms/djangoapps/ora_staff_grader/mock/data/ora_metadata.json

```
@@ -63,6 +70,13 @@
                         },
                         {
                             "orderNum": 1,
+                            "name": "good",
+                            "label": "good",
```

Similar capitalization note here

```suggestion
                            "name": "good",
                            "label": "Good",
```